### PR TITLE
RD-6168 Install plugins using the best available Python version

### PR DESCRIPTION
**Pull request by mateumann**
Because of the comments in https://github.com/cloudify-cosmo/cloudify-common/pull/1162

### CODE REVIEW

**Review by tehasdf — APPROVED** [review]

File: cloudify/plugin_installer.py [hunk 1]
```diff
@@ -562,12 +562,19 @@ def get_pth_dir(venv=None):
...
+        # This is the case for source plugins
         return sys.executable
-    if 'py310' in plugin['supported_py_versions']:
+
+    v = sys.version_info
+    if f'py{v.major}{v.minor}' in plugin['supported_py_versions']:
         return sys.executable
-    if 'py36' in plugin['supported_py_versions']:
-        return '/usr/bin/python3.6'
+    for v in reversed(plugin['supported_py_versions']):
```
Comment (geokala) [inline cloudify/plugin_installer.py:571]: What does the content of plugin['supported_py_versions'] look like? Is it a string?

Comment (tehasdf) [reply cloudify/plugin_installer.py:571]: it's a list of `["py36", "py310"]`. Yes, without the dot. It is stupid, yes.

Comment (tehasdf) [reply cloudify/plugin_installer.py:571]: cf https://github.com/cloudify-cosmo/wagon/blob/4019fbc28e915da5b522abcdaefece2493409a4d/wagon.py#L417

Comment (mateumann) [reply cloudify/plugin_installer.py:571]: no, it's a list like `['py36']`

Comment (geokala) [reply cloudify/plugin_installer.py:571]: OK, makes sense then- lgtm.

**Review by geokala — APPROVED** [review]

**Pull request merged**